charts/newrelic-infrastructure-v3/values.yaml

```diff
@@ -24,8 +24,8 @@ images:
   # Image for the New Relic Kubernetes integration.
   integration:
     pullPolicy: IfNotPresent
-    # -- Tag for the kubernetes integration.
-    tag: 0.1.2
+    # -- Tag for the kubernetes integration. Currently, set from the appVersion.
+    tag: ""
```

Comment:
Are we sure that this is going to be parsed by renovate? @roobre

Renovate by default will update [helm dependencies](https://docs.renovatebot.com/modules/manager/helmv3/) from `Chart.yaml` and [image version](https://docs.renovatebot.com/modules/manager/helm-values/) from `values.yaml`

I don't know if this change will need further configuration.

Reply:
I did not think to renovate honestly 😓 
I was just annoyed to have 3 places to update a version -> Readme, values, appVersion in chart.yaml...

If renovate takes care of that I am more than happy to rollback the change 😄

Reply:
Or nevertheless I can take it out, it it outofscope with respect to the PR

Reply:
Renovate should indeed take care, I'd keep the tag in there :)
If we see that the `appVersion` is not handled by it we can then think about something on that regard.